ui/src/views/404/style.js.scss

```diff
@@ -1,9 +1,45 @@
-message {
-    color: hotpink;
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    height: 100%;
-    font-size: 50vw;
-    line-height: 0;
+fluidWrap {
+    max-width: 940px;
+    margin: 0 auto;
+    padding: 0 10px;
+}
+
+topbar {
+    background: #ededed;
+    float: left;
+    font-size: 12px;
+    margin-bottom: 20px;
+    width: 100%;
+    border-bottom: 1px solid #c6c6c6;
+}
+
+heading {
+    font-size: 24px;
+    line-height: 30px;
+    font-weight: normal;
+    font-family: georgia, serif;
+    margin: 10px 0 0;
+}
+
+subHeading {
+    line-height: 20px;
+    font-size: 18px;
+    font-weight: normal;
+    margin: 0;
+}
+
+subHeadingContainer {
+    border-top-width: 3px;
+    border-color: #0061a6;
+    border-top-style: solid;
+    background-color: #ededed;
+    padding: 4px 5px 14px;
+    margin-bottom: 10px;
+    clear: both;
+}
+
+para {
+    margin: 0 0 10px;
+    font-family: arial;
+    font-size: 12px;
 }
```

Comment:
Which order though? Alphabetical would cause the least brain work / bikeshedding. Unless there's a way of automatically correcting the order during linting.